cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -135,6 +135,10 @@ type AutoscalingOptions struct {
 	// Setting it to false employs a more lenient filtering approach that does not try to pack the pods on the nodes.
 	// Pods with nominatedNodeName set are always filtered out.
 	FilterOutSchedulablePodsUsesPacking bool
+	// FilterOutSchedulablePodsInUpcomingNodes if set, will prevent CA from considering pods that can
```

Comment:
Seems like `AgainsUpcomingNodes` would be better than `InUpcomingNodes`

Reply:
Solved.